hedera-node/data/config/bootstrap.properties

```diff
@@ -1,2 +1,3 @@
 bootstrap.ledger.nodeAccounts.initialBalance=100000000000000
 bootstrap.ledger.systemAccounts.initialBalance=100000000000000
+ledger.keepRecordsInState=true
```

Comment:
Per Nosh's comment in scrum, it seems we don't want this to override to be the default in `master`.

Reply:
Done removing it. Thanks!